### PR TITLE
[Service provider] 'Адміністрування' tab - action buttons

### PR DESCRIPTION
**Pull request by perepichai**
Implement the action buttons and status column (UI and logic)

image

### CODE REVIEW

**Comment by litvinets** [comment]
maybe I am wrong, but right now routing doesn't work, so I cannot go to the pages for review

**Review by litvinets — CHANGES_REQUESTED** [review]
routing doesn't work

File: src/app/shell/personal-cabinet/provider-admins/provider-admins.component.ts [hunk 1]
```diff
@@ -76,7 +92,7 @@ export class ProviderAdminsComponent implements OnInit, OnDestroy {
       takeUntil(this.destroy$))
       .subscribe((params: Params) => {
         this.filteredProviderAdmins = this.providerAdmins.filter(
-          (user) => user.deputy === providerAdminRoleUkr[params.param]
+          (user) => user.isDeputy === !!(params.param === 'deputy')
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/provider-admins/provider-admins.component.ts:95]: no need to use !!. comparison operator returns boolean

**Review by dmitryy90 — APPROVED** [review]

File: src/app/shell/personal-cabinet/provider-admins/provider-admins.component.ts [hunk 2]
```diff
@@ -65,7 +74,14 @@ export class ProviderAdminsComponent implements OnInit, OnDestroy {
...
-      })
+      });
+
+      this.provider$.pipe(
+        filter((provider: Provider) => !!provider),
+        takeUntil(this.destroy$)
+      ).subscribe((provider: Provider) => {
+        this.provider = provider;
+      });
   }
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/provider-admins/provider-admins.component.ts:85]: .subscribe((provider: Provider) => this.provider = provider);

**Comment by litvinets** [comment]
sorting for other tabs doesn't work 
image

**Comment by litvinets** [comment]
image
seems like here is missing something

**Comment by litvinets** [comment]
I believe that for director it should either have to second step i the form, or add workshops dropdown as well

**Review by litvinets — APPROVED** [review]

**Pull request merged**